### ISSUE TITLE
MakeSpriteFont/SpriteFont Graphics Quality is Poor Below Size 12

### ISSUE THREAD

**Issue by mettafort** [microsoft/DirectXTK#234]
Hi-

I have used the MakeSpriteFont with lots of different fonts on my machine, and unless I stick to size 12+, the font quality is basically unusable.  I have tried various options from the commandline, but nothing fixes the poor quality of a font size <= 11.

I am wondering if this is expected, or if there is maybe an issue somewhere?

Thanks,

mf.

**Comment by walbourn** [comment]
In general the true-type fonts have difficulty scaling to really small sizes. Raster fonts seem to work best for extremely small text, which is why I have [this issue](https://github.com/microsoft/DirectXTK/issues/39) open.

Can you provide an example command-line for a case you feel is problematic?

**Comment by mettafort** [comment]
Hi Chuck-

I have attached an image of Segoe UI at size 11 (which isnt too small), and I used the following command line:

MakeSpriteFont "segoe ui" segoe_ui_11.spritefont /FontSize:11 /Sharp

![MakeSprite Example](https://user-images.githubusercontent.com/15719101/93942571-4a7bc080-fcf6-11ea-9d00-072b8d326c5d.png)

I am new to DirectX so this could be a misunderstanding on my part - I hope thats the case then I can get this fdont or something similar to work at this size Chuck.

Thanks,

mf.

**Comment by walbourn** [comment]
If you look at the generated sprite-sheet. it has the full character captured. I don't think the issue is with the font.

```
MakeSpriteFont "segoe ui" segoe_ui_11.spritefont /FontSize:11 /Sharp /DebugOutputSpriteSheet:test.bmp
```

What is your code for drawing? It looks to me like you may not have the clip-scissors set...  Remember that with descenders, the font is not "11 pixels" high but "11 points". The line spacing is 19.5 pixels so you need a little more space.

**Comment by mettafort** [comment]
Hi-

Thanks for the response.  I did not see the Debug commandline option.

I looked into the clip-scissors and I had it set to true but I dont set a rectangle.  I turned that off.

Looking through these objects I think I spotted the problem.  I am creating my render target view and viewport from the window dimensions, not the windows client dimensions - when I fix that, the font looks a lot nicer now Chuck.

It`s a shame the MakeSpriteFont doesnt include the '.' (period), since I need that to show a decimal number to the screen.  Other than that problem, the fonts are definitely looking much better.

I appreciate your patience and time with this, thank you.  

mf.
